shell/platform/windows/window_win32.h

```diff
@@ -199,6 +199,8 @@ class WindowWin32 : public KeyboardManagerWin32::WindowDelegate {
 
   UINT GetCurrentHeight();
 
+  virtual float GetScrollOffsetMultiplier();
```

Comment:
This needs a declaration comment, per the C++ style guide.

Reply:
Added comment